tests/build/test_build_individual.py

```diff
@@ -155,3 +154,113 @@ def test_build_individuals(parsed_case):
         assert ind_obj['individual_id'] == ind_info['individual_id']
         assert ind_obj['display_name'] == ind_info['display_name']
         assert ind_obj['capture_kits'] == ind_info['capture_kits']
+
+# Cancer stuff
+
+def test_build_individual_tmb():
+    ## GIVEN information about a individual
+    ind_info = {
```

Comment:
It's a lot of ind_info dictionaries here. Could it be moved into the fixtures and imported from there? 
An even cooler alternative would be to read from the real demo conf file and do the case load as one would do from the cli. But I don't know if this would add a lot of complexity. In that case nevermind!

Reply:
They are so different so I do not see any value of using a fixture...